### ISSUE TITLE
Bugs with counting total messages processed

### ISSUE THREAD

**Issue by bendavies** [zenstruck/messenger-monitor-bundle#73]
Hi Kevin,

There appears to be a problem with counting the total number of messages processed

<img width="461" alt="image" src="https://github.com/zenstruck/messenger-monitor-bundle/assets/625392/a4a43c46-64ab-4fd9-967b-9812fc22bfd4">

 because the [query to do so looks like this](https://github.com/zenstruck/messenger-monitor-bundle/blob/a6e499f0bf0bfb9835cfc37592553c40d4716209/src/History/Storage/ORMStorage.php#L105):

```sql
  SELECT COUNT(p0_.finished_at) AS sclr_0 
    FROM processed_messages p0_ 
   WHERE p0_.finished_at >= ? 
     AND p0_.failure_type IS NOT NULL 
ORDER BY p0_.finished_at DESC
```

This query does not return a single count but instead a row for each finished message.

the correct query should be:

```sql
  SELECT COUNT(*) AS sclr_0 
    FROM processed_messages p0_ 
   WHERE p0_.finished_at >= ?
     AND p0_.failure_type IS NOT NULL;
```

Easy fix, but I'm having trouble seeing what the best fix is because of the use of the specification pattern.
How would you like this fixed?

Thanks!

**Comment by bendavies** [comment]
think this was broken by https://github.com/zenstruck/messenger-monitor-bundle/commit/a4fc554d0e9a793f404e5f18767f7f4d06258d6d

**Comment by kbond** [comment]
I think this was fixed by #84. Let me know if it wasn't.
